src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-sync-metadata-info/rgw-sync-metadata-info.component.html

```diff
@@ -1,55 +1,55 @@
 <span *ngIf="metadataSyncInfo === 'no sync (zone is master)'">
   <ul class="me-2">
-    <li><b>Status:</b></li>
+    <li><b i18n>Status:</b></li>
     <li>No Sync</li>
   </ul>
 </span>
 <span *ngIf="metadataSyncInfo !== 'no sync (zone is master)'">
-  <ng-template #metadataSyncPopover>
-    <ul class="text-center">
-      <li><h5><b>Metadata Sync Status:</b></h5></li>
-      <li *ngFor="let status of metadataSyncInfo.fullSyncStatus">
-        <span *ngIf="!status?.includes(metadataSyncInfo.syncstatus) && !status?.includes('failed') && !status?.includes('error')">
-          <span *ngIf="status?.includes(':')">
-            <b>{{ status.split(':')[0] | titlecase }}</b>:{{ status.split(':')[1] | titlecase}}
-          </span>
-          <span *ngIf="!status?.includes(':')">
-            <b>{{ status | titlecase }}</b>
-          </span>
-        </span>
-        <span *ngIf="status?.includes('failed') || status?.includes('error')">
-            {{ status | titlecase }}
-        </span>
-      </li>
-    </ul>
-  </ng-template>
   <ul class="me-2">
     <ng-template #upToDateTpl>
-      <li class="badge badge-success">Up to Date</li>
+      <li class="badge badge-success" i18n>Up to Date</li>
     </ng-template>
-    <ng-template #showMetadataStatus>
-      <a *ngIf="metadataSyncInfo.syncstatus !== 'Not Syncing From Zone'"
-         class="lead text-primary"
-         [ngbPopover]="metadataSyncPopover"
-         placement="top"
-         popoverClass="rgw-overview-card-popover"
-         i18n>{{ metadataSyncInfo.syncstatus | titlecase }}</a>
-      <a *ngIf="metadataSyncInfo.syncstatus === 'Not Syncing From Zone'"
-         class="lead text-primary"
-         [ngbPopover]="metadataSyncPopover"
-         placement="top"
-         popoverClass="rgw-overview-card-popover"
-         i18n>Not Syncing</a>
-    </ng-template>
-    <li><b>Status:</b></li>
-    <li *ngIf="metadataSyncInfo.syncstatus?.includes('failed') || metadataSyncInfo.syncstatus?.includes('error'); else showMetadataStatus">
-      <i class="text-danger"
-         [ngClass]="[icons.danger]"></i>
-      <a class="lead text-danger"
-         [ngbPopover]="metadataSyncPopover"
-         placement="top"
-         popoverClass="rgw-overview-card-popover"
-         i18n>Error</a></li>
+    <cds-toggletip  [dropShadow]="true"
+                    placement="top"
+                    [align]="align">
```

Comment:
btw I see that the toggletip supports `autoAlign`. Have you tried that? https://angular.carbondesignsystem.com/?path=/story/components-toggletip--with-auto-align
```suggestion
                    align="top">
```

Reply:
I just @aaSharma14 's comment on why this shouldn't be autoAlign: https://github.com/ceph/ceph/pull/60351#pullrequestreview-2488324224. Sorry for the confusion @prgoel-code , but this can be align=top as @aaSharma14 pointed out.